brave-snouts-frontend/src/app/shared/support/support.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-support',
+  templateUrl: './support.component.html',
+  styleUrls: ['./support.component.scss']
+})
+export class SupportComponent implements OnInit {
+
+  constructor() { }
```

Comment:
missing dialog injection. Check out angular material docs about that 
You'll need to it to call `close` method